recipes/scikit-multiflow/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "scikit-multiflow" %}
+{% set version = "0.1.4" %}
+{% set sha256 = "69fa62479d6b4cf3aa04800fde3c1cf012c3350fff1cc6f5648a0e646ecbfc5d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
```

Comment:
```suggestion
  number: 0
```

Reply:
Reset in f31ce0c380863ab695530ba37e0b80ed3aa3ec5d